src/yetibot/api/google.clj

```diff
@@ -0,0 +1,267 @@
+(ns yetibot.api.google
+  (:require
+   [taoensso.timbre :refer [info warn]]
+   [clojure.string :as s]
+   [clojure.data.json :as json]
+   [clj-http.client :as client]
+   [yetibot.core.config :refer [conf-valid? config-for-ns]]))
+
+(defonce api-url "https://www.googleapis.com/customsearch/v1?parameters")
+
+(defonce messages
+  {:error-message "Values has to be one of the following: "})
+
+(defonce display-order {:normal [:title :link :snippet]
+                        :image  [:title :snippet :link]})
+
+(def config (config-for-ns))
```

Comment:
I'm working on killing all `config-for-ns` usages in another branch as it doesn't reliably determine the correct namespace in all circumstances. Can you pls replace with:

```clojure
(yetibot.core.config/get-config :yetibot :api :google) ;; can refer get-config above
```
?